presto-orc/src/main/java/com/facebook/presto/orc/OrcReader.java

```diff
@@ -267,7 +269,7 @@ static void validateFile(
         }
     }
 
-    static void validateWrite(Optional<OrcWriteValidation> writeValidation, OrcDataSource orcDataSource, Predicate<OrcWriteValidation> test, String messageFormat, Object... args)
+    public static void validateWrite(Optional<OrcWriteValidation> writeValidation, OrcDataSource orcDataSource, Predicate<OrcWriteValidation> test, String messageFormat, Object... args)
```

Comment:
Why is this change?

Reply:
Because cache related class is moved to a separate package and this is used in `StorageOrcFileTailSource`.